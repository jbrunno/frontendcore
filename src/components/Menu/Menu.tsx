import React from 'react';
import { Menu as MuiMenu, MenuProps as MuiMenuProps } from '@mui/material';

export type MenuProps = MuiMenuProps;

export function Menu(props: MenuProps) {
  return <MuiMenu {...props} />;
}
export default Menu;
