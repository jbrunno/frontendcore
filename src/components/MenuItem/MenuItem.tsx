import React from 'react';
import {
  MenuItem as MuiMenuItem,
  MenuItemProps as MuiMenuItemProps,
} from '@mui/material';

export type MenuItemProps = MuiMenuItemProps;

export function MenuItem(props: MenuItemProps) {
  return <MuiMenuItem {...props} />;
}
export default MenuItem;
