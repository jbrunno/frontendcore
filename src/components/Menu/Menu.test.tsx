import React from 'react';
import { render } from '@test-utils/component-testing';
import { Menu } from './Menu';
import MenuItem from '../MenuItem/MenuItem';

test('renders the Menu', () => {
  const anchor = document.createElement('div');
  const { container } = render(
    <Menu open anchorEl={anchor}>
      <MenuItem>Item</MenuItem>
    </Menu>,
  );
  expect(container).toMatchSnapshot();
});
