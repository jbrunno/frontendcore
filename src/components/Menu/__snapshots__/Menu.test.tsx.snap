// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Menu 1`] = `
<div
  aria-hidden="true"
/>
`;
